src/element/sequence.rs

```diff
@@ -39,6 +39,12 @@ impl Sequence {
     }
 }
 
+impl AsRef<Sequence> for Sequence {
```

Comment:
🗺️ Allows methods that take an `AsRef<Sequence>` to accept a `List`, `SExp`, or `Sequence`.